ddtrace/tracer.py

```diff
@@ -210,6 +211,9 @@ def __init__(
 
         self._new_process = False
 
+        if asbool(os.getenv("DD_TRACE_FLARE_ENABLED", default=False)):
+            self._flare = TracerFlare(self)
```

Comment:
@bengl had a really great idea that we could setup a signal handler as a trigger to generate this.

e.g. 

```
kill -SIGUSR2 <pid of python app>
```

(not sure the best signal to use, SIGUSR1 or SIGUSR2 seem like good candidates?)